src/utils/loadBrain.ts

```diff
@@ -1,23 +1,23 @@
-import { apis } from '@/brain/apis';
-import { appHome } from '@/brain/appHome';
-import { ghaCancel } from '@/brain/ghaCancel';
-import { githubMetrics } from '@/brain/githubMetrics';
-import { gocdConsecutiveUnsuccessfulAlert } from '@/brain/gocdConsecutiveUnsuccessfulAlert';
-import { gocdDataDog } from '@/brain/gocdDataDog';
-import { gocdNoDeploysAlert } from '@/brain/gocdNoDeploysAlert';
-import { gocdSlackFeeds } from '@/brain/gocdSlackFeeds';
-import { issueLabelHandler } from '@/brain/issueLabelHandler';
-import { issueNotifier } from '@/brain/issueNotifier';
-import { notificationPreferences } from '@/brain/notificationPreferences';
-import { notifyOnGoCDStageEvent } from '@/brain/notifyOnGoCDStageEvent';
-import { pleaseDeployNotifier } from '@/brain/pleaseDeployNotifier';
-import { projectsHandler } from '@/brain/projectsHandler';
-import { requiredChecks } from '@/brain/requiredChecks';
-import { saveGoCDStageEvents } from '@/brain/saveGoCDStageEvents';
-import { syncSlackUsers } from '@/brain/syncSlackUsers';
-import { syncUserProfileChange } from '@/brain/syncUserProfileChange';
-import { triggerPubSub } from '@/brain/triggerPubSub';
-import { typescript } from '@/brain/typescript';
+import { apis } from '@/brain/github/apis';
+import { ghaCancel } from '@/brain/github/ghaCancel';
+import { githubMetrics } from '@/brain/github/githubMetrics';
+import { issueLabelHandler } from '@/brain/github/issueLabelHandler';
+import { issueNotifier } from '@/brain/github/issueNotifier';
+import { pleaseDeployNotifier } from '@/brain/github/pleaseDeployNotifier';
```

Comment:
This is a bit of a weird one since it fits into both github and gocd, though I do think it fits better in the former as you have it. Any thoughts?

Reply:
The idea is to group the listeners for the upstream sources together, so I think github is right here